lib/deploy/events/apiGateway/methods.js

```diff
@@ -234,6 +241,12 @@ module.exports = {
         http.request.template
       );
     }
+
+    if (_.has(http, 'action')) {
+      // no template if some action was defined.
+      return {};
```

Comment:
Does this mean any user-specified request template won't be used? e.g. if someone wants to use a GET to describe the execution and pass in the execution ARN as a query string param, would they be able to write a custom request template to do that?

Reply:
No, if user specify http.request.template option, it will be applied. These codes are just avobe `return defaultTemplate`. The default action for StartExecution should not be applied all other actions and should pass users action through.

Reply:
right, right, I see. Thanks, that looks good to me.